honeybadgermpc/offline_randousha.py

```diff
@@ -136,6 +137,55 @@ def get_degree_and_secret(shares):
     return tuple(zip(out_t, out_2t))
 
 
+async def generate_triples(n, t, k, my_id, _send, _recv, field):
+    subscribe_recv_task, subscribe = subscribe_recv(_recv)
+
+    def _get_send_recv(tag):
+        return wrap_send(tag, _send), subscribe(tag)
+
+    # Start listening for my share of t and 2t shares from all parties.
+    send, recv = _get_send_recv("randousha")
+    rs_t2t = await randousha(n, t, 3 * k, my_id, send, recv, field)
+
+    as_t2t = rs_t2t[0*k:1*k]
+    bs_t2t = rs_t2t[1*k:2*k]
+    rs_t2t = rs_t2t[2*k:3*k]
+
+    as_t, _ = zip(*as_t2t)
+    bs_t, _ = zip(*bs_t2t)
+    rs_t, rs_2t = zip(*rs_t2t)
+
+    # Config just has to specify mixins used by switching_network
+    config = {}
+
+    # Compute degree reduction to get triples
+    # TODO: Use the mixins and preprocessing system
+    async def prog(ctx):
+        abrs_2t = [a * b + r for a, b, r in zip(as_t, bs_t, rs_2t)]
+        assert len(rs_2t) == len(rs_t) == len(as_t) == len(bs_t)
+        abrs = await ctx.ShareArray(abrs_2t, 2*t).open()
+        # abrs = await ctx.ShareArray(rs_t).open()
```

Comment:
Could you clean out the commented code or provide a comment saying why they're commented?